pyEpiabm/pyEpiabm/tests/test_unit/test_property/test_personal_foi.py

```diff
@@ -26,15 +26,14 @@ def setUpClass(cls) -> None:
 
     def test_person_inf(self):
         result = PersonalInfection.person_inf(self.infector, self.time)
-        self.assertTrue(result > 0)
+        self.assertEqual(result, 1)
         self.assertIsInstance(result, float)
-        self.assertEqual(result, self.infector.infectiousness)
 
     def test_person_susc(self):
         result = PersonalInfection.person_susc(self.infector,
                                                self.infectee,
                                                self.time)
-        self.assertTrue(result > 0)
+        self.assertEqual(result, 1.0)
```

Comment:
Tiny thing, but is there a reason why the person_inf is 1 but the person_susc is 1.0?

Reply:
Nope, they're both floats so the one is really 1.0. I will change for clarity.